src/st/no3/3FF00.c

```diff
@@ -0,0 +1,1208 @@
+#include "no3.h"
+
+void EntityRoomTransition2(Entity* self) {
+    Entity* newEntity;
+    Primitive* prim;
+    // Used as both a loop variable and a temp
+    s32 localVar;
+
+    // These are weird fake vars needed for matching
+    Entity* gents = g_Entities;
+    Unkstruct8* layout = &g_CurrentRoomTileLayout;
```

Comment:
These are not fake, it's a common pattern for entity functions that use g_CurrentRoomTileLaout.